docs/genai/tutorials/phi3-v.md

```diff
@@ -91,6 +88,7 @@ Support for Windows machines with GPUs other than NVIDIA is coming soon!
 
    ```bash
    curl https://raw.githubusercontent.com/microsoft/onnxruntime-genai/main/examples/python/phi3v.py -o phi3v.py
+   pip install pyreadline3
```

Comment:
Why is `pyreadline3` required for the tutorial?

Reply:
It's used to read the image file location from the command line